### ISSUE TITLE
Set up a DB to store game data

### ISSUE THREAD

**Issue by adrian-golian** [Blef-team/blef_game_engine#67]
Choose a DB, migrate storage schema from RDS and deploy on EC2.

**Comment by maciej-pomykala** [comment]
Many companies nowadays use flat, cross-platform files such as CSV. However, there is no one-size-fits-all approach and in this team we're fans of NoCSV.

**Comment by maciej-pomykala** [comment]
No longer relevant after #169
